testing/unity/Makefile

```diff
@@ -68,7 +68,6 @@ $(UNITY_TARBALL):
 $(UNITY_UNPACKNAME): $(UNITY_TARBALL)
 	@echo "Unpacking: $(UNITY_TARBALL) -> $(UNITY_UNPACKNAME)"
 	$(Q) $(UNPACK) $(UNITY_TARBALL)
-	@touch $(UNITY_UNPACKNAME)
```

Comment:
This should stay too, unzip or tar will keep the timestamps of what's inside the archive.

Reply:
Ok, keep touch for unzip too.